packages/edit-site/src/components/sidebar-navigation-screen-templates/style.scss

```diff
@@ -1,4 +1,11 @@
-.edit-site-sidebar-navigation-screen-templates__see-all {
-	/* Overrides the margin that comes from the Item component */
-	margin-top: $grid-unit-20 !important;
+.edit-site-sidebar-navigation-screen-templates__templates-list {
```

Comment:
would be nice to create a more generic class for this overflow scroll behaviour and styling so that it can be applied to any list